issue-triaging.md

```diff
@@ -44,6 +44,52 @@ between the involved parties to allow work to begin or for the issue to be close
 - Guiding any interested parties to another person or resource that may be more knowledgeable about an issue.
 - Suggesting an issue for discussion at a SIG meeting if a synchronous discussion would be more productive.
 
+It is recommended that a triager should follow these steps when a new issue is reported:
+
+1. First and foremost step is to access the validity of the issue. Here's an rough outline to access the validity:
+   - Does the issue include a clear description of the problem?
+   - Does it provide the user’s configuration, the expected behavior, and any other relevant context?
+   - If it's reported as a bug, are the steps to reproduce clearly outlined?
+
+2. If the issue is deemed valid and requires attention, apply appropriate labels based on its nature:
+   - For bugs, add the `bug` label along with `waiting-for-codeowners`.
+   - For enhancements, use the `enhancement` label and also add `waiting-for-codeowners`.
+   - If the issue is about a new component, add the `sponsor-needed` label. We should encourage user to join our SIG and propose the new component. They can also reach out on #otel-collector on CNCF slack.
```

Comment:
```suggestion
   - If the issue is about a new component, add the `sponsor-needed` label. We should encourage the user to join our SIG meeting and propose the new component to a broader audience. They can also reach out in the #otel-collector-dev channel on CNCF Slack.
```
Nits: Adjusted the wording as filing an issue for a new component is the same as "propose the new component". Bringing it up in the sig meeting is to make a wider audience aware of the proposal in hopes of more feedback, interest, and finding a sponsor.

Also, I could go either way on `#otel-collector` vs `#otel-collector-dev`.